Formula/hashcat.rb

```diff
@@ -0,0 +1,21 @@
+class Hashcat < Formula
+  desc "World's fastest and most advanced password recovery utility"
+  homepage "https://hashcat.net/hashcat/"
+  url "https://hashcat.net/files/hashcat-#{version}.tar.gz"
+  version "3.10"
```

Comment:
Plus the `version` is nil when `"https://hashcat.net/files/hashcat-#{version}.tar.gz"` is parsed; you’d need to put the `version "3.10"` line before it.